lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -129,6 +129,41 @@ def test_global_scatter_wrap_no_transform():
     plt.scatter(x, y, c=data)
 
 
+@pytest.mark.natural_earth
+@ImageTesting(['global_hexbin_wrap'],
+              tolerance=27.6 if MPL_VERSION < '2.1.0' else 0.5)
+def test_global_hexbin_wrap():
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    ax.coastlines(zorder=2)
+    x, y = np.meshgrid(np.arange(-179.9, 180, 1.), np.arange(-90, 91))
+    data = np.sin(np.sqrt(x**2 + y**2))
+    plt.hexbin(
+        x.flatten(),
+        y.flatten(),
+        C=data.flatten(),
+        gridsize=20,
+        zorder=1,
+    )
+
+
+@pytest.mark.natural_earth
+@ImageTesting(['global_hexbin_wrap'],
+              tolerance=27.6 if MPL_VERSION < '2.1.0' else 0.5)
+def test_global_hexbin_wrap_transform():
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    ax.coastlines(zorder=2)
+    x, y = np.meshgrid(np.arange(0.1, 360, 1.), np.arange(-90, 91))
+    x_wrap = np.where(x >= 180, x-360, x)
```

Comment:
Put a short comment here indicating why this wrap is needed for the data to be the same.

Reply:
very good, thank you.